packages/components/pie-spinner/README.md

```diff
@@ -0,0 +1,73 @@
+<p align="center">
+  <img align="center" src="../../../readme_image.png" height="200" alt="">
+</p>
+
+<p align="center">
+  <a href="https://www.npmjs.com/@justeattakeaway/pie-spinner">
+    <img alt="GitHub Workflow Status" src="https://img.shields.io/npm/v/@justeattakeaway/pie-spinner.svg">
+  </a>
+</p>
+
+# Table of Contents
+
+1. [Introduction](#pie-spinner)
+2. [Installation](#installation)
+3. [Importing the component](#importing-the-component)
+4. [Peer Dependencies](#peer-dependencies)
+5. [Props](#props)
+6. [Testing](#testing)
```

Comment:
Do we need a section for this or does the generator need updating?

Reply:
i will check this in a follow up PR